test/helpers/pkcs12.c

```diff
@@ -358,6 +360,11 @@ static int add_attributes(PKCS12_SAFEBAG *bag, const PKCS12_ATTR *attrs)
             if (!TEST_true(PKCS12_add_localkeyid(bag, (unsigned char *)p_attr->value,
                                                  strlen(p_attr->value))))
                 goto err;
+        } else if (attr_nid == NID_oracle_jdk_trustedkeyusage) {
+            attrs = (STACK_OF(X509_ATTRIBUTE)*)PKCS12_SAFEBAG_get0_attrs(bag);
+            x509_attr = X509_ATTRIBUTE_create(attr_nid, V_ASN1_OBJECT, OBJ_txt2obj(p_attr->value, 0));
+            X509at_add1_attr(&attrs, x509_attr);
+            PKCS12_SAFEBAG_set0_attrs(bag, attrs);
```

Comment:
There's a mem leak here. The `1` in `X509at_add1_attr()` means that ownership of the object remains with the caller, so the contents of `x509_attr` needs to be freed. Add a call to `X509_ATTRIBUTE_free(x509_attr)`. This looks like the cause of the CI failures.

Reply:
Ah right OK, I've removed that code anyway as it wasn't needed and I'm trying to test the set0_attrs in isolation as opposed to within that existing add_attributes()